MANIFEST.in

```diff
@@ -1,8 +1,9 @@
-include *.in
-include *.py
+prune .github
 include *.md
-include LICENSE
 include tox.ini
-include pyproject.toml
```

Comment:
These both seem like good things to include in the sdist. Are they included automatically now?

Reply:
As per #synapse-dev, it looks like `python -m build` always includes all files from the repository by default when run on my machine, not sure why.

According to https://packaging.python.org/en/latest/guides/using-manifest-in/, `pyproject.toml` is automatically included now. I've also added the license file to `license_file` in `setup.cfg` since it seems like that will always cause it to be included.

Reply:
I've now replaced the content of `MANIFEST.in` with the suggestion from `check-manifest`. The only exception to this is that I've added a `prune .github` to remove GitHub codeowner and workflow files, which felt to me like they shouldn't live in the sdist.